app/styles/headers/_eighth.scss

```diff
@@ -0,0 +1,147 @@
+.header__eighth{
+    background-image: url(../img/eighth.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    padding-top: 48px;
+    padding-bottom: 90px;
+    color: #fff;
+    font-family: 'Cabin', arial;
+    font-weight: 400;
+    height: auto;
+    font-size: 16px;
+
+    a{
+        color: unset;
+        text-decoration: none;
+    }
+
+    .container{
+        width: 940px;
+        max-width: 100%;
+        padding: 0 20px;
+        margin: 0 auto;
+    }
+
+    .action-btn{
+        margin-top: 75px;
+    }
+
+    .button{
+        color: #fff;
+        border-radius: 60px;
+        height: 60px;
+        border: none;
+        background-color: #db7560;
+        padding: 0 72px;
+        transition: 0.2s;
+        cursor: pointer;
+        font-size: 18px;
+
+        &:hover{
+            transform: scale(1.1);
+            background-color: #f78d78;
+        }
+    }
+
+    .nav{
+        display: flex;
+        justify-content: space-between;
+        color: #383737;
+        background-color: #f2f5f8;
+        padding: 16px 20px;
+        border-radius: 4px;
+        max-width: 100%;
+        font-weight: 600;
+
+        &__item{
+            margin-left: 40px;
+            display: flex;
+            align-items: center;
+            transition: transform 0.2s;
+
+            &:first-child{
+                margin-left: 0;
+            }
+
+            &:hover{
+                transform: scale(1.1);
+            }
+        }
+
+        &__search-form{
+            display: flex;
+            align-items: center;
+        }
+    }
+
+    .form-group{
+        color: #c6ccd2;
```

Comment:
please, move this color to variable and use variable here and in the other places